python/cudf/cudf/tests/test_multiindex.py

```diff
@@ -428,17 +428,15 @@ def test_multiindex_loc_rows_1_2_key(pdf, gdf, pdfIndex):
     gdfIndex = cudf.from_pandas(pdfIndex)
     pdf.index = pdfIndex
     gdf.index = gdfIndex
-    print(pdf.loc[("c", "forest"), :])
-    print(gdf.loc[("c", "forest"), :].to_pandas())
+
```

Comment:
Good finds. I opened a PR https://github.com/rapidsai/cudf/pull/18217 to add a Ruff rule so we avoid `print`s in general

Reply:
Thanks @mroeschke !